playground/backend/internal/storage/cloud_storage.go

```diff
@@ -0,0 +1,218 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
```

Comment:
regarding the package name, it's unclear actually what functionality it's provide.  Idiomatically in go package name should tell us which functionality it provides, i.g.` encoding/json` say that we will work with encoding for  JSONs. In our cases `validators/java_validator` it's clear too. Moreover, we also had a similarly named package in cache PR's before Pablo ask to rename it because it looked unclear.
I suppose to name it somehow like: `precompiled_objects/public_bucket`